src/components/AdjustVaultForm.tsx

```diff
@@ -42,21 +91,28 @@ const AdjustVaultForm = () => {
         <div className="col-span-2 lg:col-span-1">
           <div className="mb-6">
             <StyledDropdown
-              selection="No Action"
-              choices={['No Action']}
-              onSelection={() => {
-                // TODO
-                console.log('Handle Action Selected');
-              }}
+              selection={debtAction}
+              choices={[DebtAction.None, DebtAction.Borrow, DebtAction.Repay]}
+              onSelection={setDebtAction}
               label="Action"
             />
           </div>
-          <AmountInput disabled label="Available Balance" />
           <AmountInput
-            onChange={() => {
-              console.log('Handle Amount Change');
-            }}
+            disabled
+            label="Available Purse Balance"
+            value={
+              (debtPurse?.currentAmount as Amount<'nat'> | undefined)?.value
+            }
+            brand={debtPurse?.brand}
+            suffix={displayBrandPetname(vaultToAdjust?.totalDebt.brand)}
+          />
+          <AmountInput
+            onChange={setDebtDeltaValue}
+            value={debtDeltaValue}
```

Comment:
`debtDeltaValue` sounds like it could go negative. But an `AmountInput` can't because an Amount can't be negative (and thankfully its `min: 0` enforces that.

If the value is a Nat we should call it that and wrap `Nat()` liberally. But even better, let's pass around the `Amount` type so it's fully enforced and we're immune to brand matching errors.

Reply:
Hmm yea it can't and shouldn't ever be negative. Maybe `debtDeltaMagnitude`? I tried to import @endo/Nat but it doesn't export types, so wasn't helpful really. I'd rather not change this to an Amount because it can't be nullable then

Reply:
Since it can't be negative, why call it "delta"? Consider `debtGain`.

Re: @endo/nat there's no expressible type for "non-zero bigint" so `Nat()` always returns bigint, but because it throws on <0 it guarantees that the number is >=0n.

Reply:
The value itself can't be negative, but in effect you could either be gaining or losing debt based on the currently selected `DebtAction`

Reply:
ah. then yeah I think `debtDeltaMagnitude` or `debtChange` would be clearer. even `debtDiff`